raven/src/main/java/com/getsentry/raven/DefaultRavenFactory.java

```diff
@@ -145,6 +142,11 @@
      */
     public static final long ASYNC_SHUTDOWN_TIMEOUT_DEFAULT = TimeUnit.SECONDS.toMillis(1);
     /**
+     * Option for which package prefixes are part of the user's application code, as a single
+     * comma separated string.
+     */
+    public static final String IN_APP_FRAMES_OPTION = "raven.stacktrace.packages";
```

Comment:
This is a little generic — maybe `app-packages` or something (whatever delimiter is stylistically consistent with the other parameters, I forget/didn't look?)